### PR TITLE
3.7.0

### PR DESCRIPTION
**Pull request by vikasyadavbtech**
Hi Editor, 
I have download and used your Family CMS script from your website. But, when I install it on my localhost, it give me a fatal error:

Fatal error: Maximum execution time of 30 seconds exceeded in C:\xampp\htdocs\PhpProject1\inc\install_inc.php on line 700

I have check the link 700 in my localhost, but no execution time is found. Please tell me how i can solve this issue.

Thanks & Regards...
vikasyadavbtech@gmail.com

### CODE REVIEW

**Comment by ryanhowdy** [comment]
Was this done on version 3.7.0 because it is still in development.

File: familyconnections/admin/debug.php [hunk 1]
```diff
@@ -0,0 +1,380 @@
...
+                                <td>'.T_('PHP Expose PHP').'</td>
+                                <td>'.(ini_get('expose_php') ? T_('On') : T_('Off')).'</td>
+                            </tr>
+                            <tr>
+                                <td>'.T_('PHP Register Globals').'</td>
+                                <td>'.(ini_get('register_globals') ? T_('On') : T_('Off')).'</td>
+                            </tr>
+                            <tr>
+                                <td>'.T_('PHP Max Script Execution Time').'</td>
+                                <td>'.(ini_get('max_execution_time') ? sprintf(T_('%s seconds'), ini_get('max_execution_time')) : T_('N/A')).'</td>
```
Comment (vikasyadavbtech) [inline familyconnections/admin/debug.php]: Please tell me the php file name where the PHP Max Script Execution Time is located..

**Comment by ryanhowdy** [comment]
There is no code in FCMS that configures Max Execution Time.  That is a PHP configuration/server configuration thing.

**Comment by vikasyadavbtech** [comment]
ok, thanks..


Issue has been resolved.

Regards
vikas yadav

**Pull request closed without merging**